frontend/app/assets/stylesheets/spree/frontend/views/spree/products/show.scss

```diff
@@ -163,3 +163,9 @@
     line-height: font-px-to-rem(26px);
   }
 }
+
+.breadcrumb-item:last-of-type {
```

Comment:
Why are you hiding this?

Reply:
It is a part of the SD-773:
`2. Product name needs to be visible in the breadcrumbs on desktop only, but not clickable.`

Reply:
1. It doesn't work properly. On smaller desktop versions (< 992px) I cannot see the last breadcrumb.
2. Use mobile-first approach instead

Reply:
Ad. 1 It will be now displayed in views with 768px. @AleksandraKaminska will it be ok?
Ad. 2 I've changed the approach